pages/events.js

```diff
@@ -1,11 +1,25 @@
-import React from 'react'
-import Link from 'next/link'
-import Layout from '../components/Layout'
+import React from "react";
+import Link from "next/link";
+import sanity from "../lib/sanity";
 
+const query = `*[_type == "backgroundVideo"]{description, "url": video.asset->url}[0]`;
 export default class Events extends React.Component {
-  render() {
+  static async getInitialProps() {
+      return {video: await sanity.fetch(query)};
+  }
+    render() {
+        console.log ('eh', this.props)
     return (
-        <Layout><p> events </p></Layout>
-    )
+      <footer>
+        <p> he? {this.props.video.description}</p>
```

Comment:
he?

Reply:
...and we can use the Footer component here?

Reply:
is delete.